lib/cheet.rb

```diff
@@ -0,0 +1,9 @@
+class Cheet
```

Comment:
it's a small one now but we could call this file model rather than lib. The definition is quite though but usually if you have a file which is mapped to an object in a database it's a model. Something which has functionality and no reason to be saved in a database you could save in a library directory.